bears/general/RegexLintBear.py

```diff
@@ -0,0 +1,55 @@
+import re
+
+from queue import Queue
+from sarge import run, Capture
+from contextlib import suppress
+
+from bears.general.AnnotationBear import AnnotationBear
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from coalib.testing.LocalBearTestHelper import execute_bear
+
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+class RegexLintBear(LocalBear):
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {PipRequirement('regexlint', '1.6')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting'}
+
+    def run(self, filename, file, language: str):
+        """
+        Bear for linting regex through regexlint.
+
+        :param language:
+            The programming language of the file(s).
+        """
+        section = Section('')
+        section.append(Setting('language', language))
+        bear = AnnotationBear(section, Queue())
```

Comment:
This is not how `AnnotationBear` should be used, and is not how it is used by any of the other bears which use it.  There are specific docs for how to use dependency bears.

Reply:
Yep, fixing it now.